.github/workflows/unit-test.yml

```diff
@@ -48,7 +48,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: --all-targets --all-features --workspace -- -D warnings
+          args: --all-targets --all-features --workspace
```

Comment:
This is workaround IMO and doesn't solve root cause

Reply:
As discussed, this is followed up in https://github.com/golemfactory/yagna/issues/2281.